test.py

```diff
@@ -0,0 +1,64 @@
+from unittest import TestCase
+from io import BytesIO
+from os import path
+
+from PIL import Image as PilImage, ImageDraw as PilDraw
+from wand.image import Image as WandImage
+
+import dhash
+
+
+IMGDIR = path.dirname(__file__)
+
+
+def pilToWand(image, format='png'):
+    with BytesIO() as fd:
+        image.save(fd, format=format)
+        fd.seek(0)
+        return WandImage(file=fd)
+
+
+class TestDHash(TestCase):
+    def test_get_grays(self):
+        with PilImage.open(path.join(IMGDIR, 'dhash-test.jpg')) as image:
+            # get the first two rows
+            result = dhash.get_grays(image, 9, 9)[:18]
+
+        expected = [
+            93, 158, 210, 122, 93, 77, 74, 74, 77,
+            95, 117, 122, 111, 92, 74, 81, 80, 77,
+        ]  # fmt: skip
```

Comment:
We may have to make this allow variation by +/-1, see [here](https://github.com/benhoyt/dhash/commit/730ce501ab1ca68c49399e407ead8844dd932a57#diff-99b846d6215ee4329114b53913fed329569ac1f94e31f5dc5330a216d383195eR48).

Reply:
oops, misread the intent of the code.  switched to using `delta=1` which should do this